src/components/FormComponent/styles.js

```diff
@@ -19,3 +22,49 @@ export const ValidationContainer = styled.div`
 export const ValidationMessage = styled.p`
   color: ${(props) => (props.successColor ? "green" : "red")};
 `;
+
+export const styledComponents = {
+  FormContainer,
+  Input,
+  Button,
+  ValidationContainer,
+  ValidationMessage,
+};
+
+export const useStyles = makeStyles((theme) => ({
```

Comment:
o, to fajnie, ogolnie robienie themu na caly projekt jest popularne w projektach, zeby bylo wszystko spojnie i zmienialne tylko w jednym miejscu

Reply:
Done.